.gitignore

```diff
@@ -1,6 +1,12 @@
 node_modules
+jspm_packages
```

Comment:
I would put these files:
.gitignore / package / package-lock to their specific PR unless they are related to the implementation of the code